db/migrations/20200331102914_create_median_kiss_batch.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
+CREATE TABLE maker.median_kiss_batch
+(
+    id         SERIAL PRIMARY KEY,
+    log_id     BIGINT     NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    header_id  INTEGER    NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
+    address_id INTEGER    NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    msg_sender INTEGER    NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    a_length   INTEGER    NOT NULL,
+    a          TEXT ARRAY NOT NULL,
+    UNIQUE (header_id, log_id)
+);
+
+CREATE INDEX median_kiss_batch_log_index
+    ON maker.median_kiss_batch (log_id);
+CREATE INDEX median_kiss_batch_header_index
+    ON maker.median_kiss_batch (header_id);
+CREATE INDEX median_kiss_batch_address_index
+    ON maker.median_kiss_batch (address_id);
+CREATE INDEX median_kiss_batch_msg_sender_index
+    ON maker.median_kiss_batch (msg_sender);
+
+
+-- +goose Down
+DROP TABLE maker.median_kiss_batch;
```

Comment:
Its probably redundant when we `DROP` the table, but in other migrations we seem to be adding a `DROP` for each index as well in the rollback commands.

Reply:
Yeah, I've moved away from adding those statements in newer code. Like you said, they seem to be totally unnecessary.